lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -458,6 +458,34 @@ def test_valid_range_and_valid_min_valid_max_provided(self):
                 saver.check_attribute_compliance(self.container, self.data)
 
 
+class Test__cf_coord_identity(tests.IrisTest):
+    def check_call(self, coord_system, units, expected_units):
+        coord = iris.coords.DimCoord([30, 45], 'latitude', units=units,
+                                     coord_system=coord_system)
```

Comment:
@lbdreyer Ideally, to give full coverage, could you also test `longitude` also ... thanks!